Formula/rhit.rb

```diff
@@ -31,27 +29,32 @@ def install
     require "io/console"
 
     resource("testdata").stage do
+      output = ""
       PTY.spawn("#{bin}/rhit --silent-load --length 0 --color no access.log") do |r, _w, _pid|
         r.winsize = [80, 130]
-        output = r.read.gsub(/\r?\n/, "\n")
-
-        assert_match <<~EOS, output
-          33,468 hits and 405M from 2021/01/22 to 2021/01/22
-          ┌──────────┬──────┬─────┬────────────────────┐
-          │   date   │ hits │bytes│0                33K│
-          ├──────────┼──────┼─────┼────────────────────┤
-          │2021/01/22│33,468│ 405M│████████████████████│
-          └──────────┴──────┴─────┴────────────────────┘
-        EOS
-        assert_match <<~EOS, output
-          HTTP status codes:
-          ┌─────┬─────┬────┬────┐
-          │ 2xx │ 3xx │4xx │5xx │
-          ├─────┼─────┼────┼────┤
-          │79.1%│14.9%│1.2%│4.8%│
-          └─────┴─────┴────┴────┘
-        EOS
+        begin
+          r.each_line { |line| output += line.gsub(/\r?\n/, "\n") }
+        rescue Errno::EIO
+          # GNU/Linux raises EIO when read is done on closed pty slave
+        end
```

Comment:
Added changes to see how CI behaves.
On local macOS and docker Linux, the test passes.